src/Apps/Order/Routes/Respond/index.tsx

```diff
@@ -61,11 +68,17 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
   } as RespondState
 
   onContinueButtonPressed: () => void = () => {
-    const { offerValue, responseOption } = this.state
+    const { responseOption } = this.state
     this.setState({ isCommittingMutation: true }, () => {
       switch (responseOption) {
         case "COUNTER":
-          window.alert(`You decided to COUNTER with ${offerValue}.`)
+          this.createCounterOffer(Number(this.state.offerValue || "0"))
```

Comment:
We'll have to update this when we add validation 😅

Reply:
I don't think we'll forget, but will add a TODO for visibility :+1: